index.js

```diff
@@ -253,14 +269,7 @@ async function runTask(options) {
   const taskDir = options.taskDirectory;
   const workDir = options.workDirectory;
 
-  const lambdaFunctionNameMatch = lambdaArn.match(/:function:([^:]+)/);
-  if (lambdaFunctionNameMatch) {
-    log.sender = `cumulus-ecs-task/${lambdaFunctionNameMatch[1]}`;
-  }
-  else {
-    const err = new TypeError(`Unable to determine lambda function name from ${lambdaArn}`);
-    log.error(`Unable to determine lambda function name from ${lambdaArn}`, err);
-  }
+  log.sender = `cumulus-ecs-task/${getFunctionName(lambdaArn)}`;
```

Comment:
Are we losing valuable functionality by no longer catching the case in which the lambdaArn actually isn't a lambdaArn? I may be mistaken, but it appears that under the proposed implementation, any parameter passed into `getFunctionName` that doesn't begin with `arn:aws:lambda` will be returned as is.

Reply:
So the issue is that the parameter is called `lambdaArn`, but it apparently accepts a Lambda function name as well as an ARN.  We need to add the Lambda function name to the logs, though, so the two cases here are that 1) the parameter is an ARN or 2) the parameter is a simple Lambda function name already.